lib/routes/account.js

```diff
@@ -646,6 +734,7 @@ module.exports = function (
             response.verificationMethod = 'email'
             response.verificationReason = 'login'
           }
+          recordSecurityEvent() // don't block response recording db event
```

Comment:
Any objection to putting this in a similar format as [checkSecurityHistory](https://github.com/mozilla/fxa-auth-server/pull/1333/files/7fbcb7e6aa7c4f95bbfd0067beec748d7dfecf1a#diff-ab87c506e682e952973178a41792df12R398), except this would go before `createResponse`?